app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoScreen.kt

```diff
@@ -79,7 +82,7 @@ class IntroVideoScreen : BaseScreen<
   }
 
   override fun openHome() {
-    val screenKey = screenKeyProvider.keyFor<Key>(this)
+    val screenKey = screenKeyProvider.keyFor<Key>(rootView)
```

Comment:
We should not be using the screen key provider anymore after migrating to a fragment. Instead, we can directly get the `screenKey` variable from the screen. So we can remove this implementation.

Also, the commit message doesn't reflect what's happening here, we are using root view for neither inflation nor transitions here. So, please update that as well.